### PR TITLE
add note about oe_builtin set under lmp configuration

### PR DESCRIPTION
**Pull request by vanmaegima**
oe_builtin often confuses our customers as the doc is misleading about this step and this should be set under lmp configuration.

Signed-off-by: Vanessa Maegima <vanessa.maegima@foundries.io>

## Readiness

* [ ] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by vanmaegima** [comment]
It's really bad that we're duplicating information in pages like this but I couldn't drop one of the pages when reviewing the docs.

**Comment by doanac** [comment]
Docs for 54227d2f7bcda52b07d4787868877ed7417a840f are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1921/docs/artifacts/html/index.html

**Comment by mike-sul** [comment]
> oe_builtin often confuses our customers as the doc is misleading about this step and this should be set under lmp configuration.

The `oe_builtin` can be defined in both `containers.preloaded_images` and `lmp.preloaded_images` sections. The `lmp` inherits `containers` and can override it.

File: source/reference-manual/linux/preloaded-images.rst [hunk 1]
```diff
@@ -40,12 +39,23 @@ Where:
...
-  This option does not work with some of the advanced tagging cases,
-  e.g. in the case of multiple container builds using the same platform (see :ref:`ref-advanced-tagging` for more details).
+
+.. note::
+
+   There is also one additional parameter ``oe_builtin: <true|false>``:
+
+   - ``oe_builtin`` - Defines a way to accomplish preloading Apps. If the option is not defined or set to `false` (by default), then Apps are preloaded by the `assemble` run of a LmP CI build. Otherwise, Apps are preloaded during an OE build CI run. In this case, rootfs as well as a system image produced by the run includes preloaded Apps. Only `Restorable` type of Apps (default) are supported by the OE builtin preloader. This option does not work with some of the advanced tagging cases, e.g. in the case of multiple container builds using the same platform (see :ref:`ref-advanced-tagging` for more details).
+
+   When setting this option, apps are preloaded during an OE/platform build CI run, so container builds will not provide an image with preloaded apps as they do not rely on OE build steps. So for this setting, the preload configuration should be under ``lmp:``:
```
Comment (kprosise) [inline source/reference-manual/linux/preloaded-images.rst:49]:
```suggestion
   When setting this option, Apps are preloaded during an OE/platform build CI run. Container builds will not provide an image with preloaded apps as they do not rely on OE build steps. For this setting, the preload configuration should be under ``lmp:``:
```

**Review by kprosise — APPROVED** [review]

**Comment by kprosise** [comment]
I will work on shortening the user guide entry into an example and link to the reference manual for greater detail on specifics/technical details.

**Comment by vanmaegima** [comment]
Closing per Mike Sul's comment that this is incorrect. Will open another one later if needed after more customer interactions.

**Pull request closed without merging**